src/helpers/libs/web3js.js

```diff
@@ -718,54 +752,133 @@ const Web3JSCalls = {
     args: [],
   },
   eth_newFilter: {
-    exec: (provider, proto, ...args) => {
-      return new Promise((resolve, reject) =>
-        reject('EtherFlow does not YET support this method.')
-      );
+    exec: async (provider, proto, ...args) => {
+      const filter = {};
+      filter.topics = args[3]
+        ? args[3].split(',').map((x) => (x === 'null' ? null : x.split('||')))
+        : [];
+      filter.fromBlock = args[0] ? args[0] : 'latest';
+      filter.toBlock = args[1] ? args[1] : 'latest';
+      filter.address = args[2] ? args[2] : null;
+      provider.extend({
+        methods: [
+          {
+            name: 'eth_newFilter',
+            call: 'eth_newFilter',
+            params: 1,
+            inputFormatter: [null],
+          },
+        ],
+      });
+      provider.extend({
+        methods: [
+          {
+            name: 'eth_getFilterChanges',
+            call: 'eth_getFilterChanges',
+            params: 1,
+            inputFormatter: [null],
+          },
+        ],
+      });
+      const filterId = await provider.eth_newFilter(filter);
+      return provider.eth_getFilterChanges(filterId);
     },
     codeSample: (url, ...args) => {
-      return '/* Not Supported by EtherFlow yet */';
+      const filter = `const filter = {
+  ${args[0] ? "fromBlock: '" + args[0] + "'" : "fromBlock: 'latest'"},
+  ${args[1] ? "toBlock: '" + args[1] + "'" : "toBlock: 'latest'"},
+  ${args[2] ? "address: '" + args[2] + "'" : ''},
```

Comment:
```suggestion
  ${args[2] ? "address: '" + args[2] + "'," : ''}
```
same reason as above

Reply:
fixed